pygmt/src/surface.py

```diff
@@ -17,9 +17,13 @@
 
 @fmt_docstring
 @use_alias(
+    C="convergence",
     I="spacing",
+    L="limit",
     R="region",
     G="outgrid",
+    M="max_radius",
```

Comment:
Change to alphabetical order.
```suggestion
    L="limit",
    M="max_radius",
    R="region",
    G="outgrid",
```